api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Suite.java

```diff
@@ -74,6 +92,8 @@
   @Valid
   private final List<Test> tests = new ArrayList<>();
 
+  private String patternCorrelationId;
```

Comment:
I don't think it can be final. It is an optional parameter which is not set in constructor.